RecoTauTag/RecoTau/python/tools/runTauIdMVA.py

```diff
@@ -152,19 +156,19 @@ def runTauID(self):
                 if self.debug: print ("runTauID: not is_above_cmssw_version(9, 4, 4). Will update the list of available in DB samples to access 2017v1")
                 self.loadMVA_WPs_run2_2017()
 
-            self.process.rerunDiscriminationByIsolationOldDMMVArun2017v1raw = patDiscriminationByIsolationMVArun2v1raw.clone(
-                PATTauProducer = cms.InputTag('slimmedTaus'),
+            setattr(self.process,"rerunDiscriminationByIsolationOldDMMVArun2017v1raw"+self.postfix,patDiscriminationByIsolationMVArun2v1raw.clone(
```

Comment:
```suggestion
           _rerunDiscriminationByIsolationOldDMMVArun2017v1raw = "rerunDiscriminationByIsolationOldDMMVArun2017v1raw"+self.postfix
                        setattr(self.process,_rerunDiscriminationByIsolationOldDMMVArun2017v1raw,patDiscriminationByIsolationMVArun2v1raw.clone(
```
and reuse it below.
This perhaps also gives an opportunity to shorten the name for better readability